crates/build/re_remote_store_types_builder/src/bin/build_re_remote_store_types.rs

```diff
@@ -0,0 +1,38 @@
+//! This binary runs the remote store gRPC service codegen manually.
+//!
+//! It is easiest to call this using `pixi run codegen-rstore`,
+//! which will set up the necessary tools.
+
+#![allow(clippy::unwrap_used)]
+
+use camino::Utf8Path;
+
+const PROTOBUF_DEFINITIONS_DIR_PATH: &str = "crates/store/re_remote_store_types/proto";
+const PROTOBUF_REMOTE_STORE_V0_RELATIVE_PATH: &str = "rerun/v0/remote_store.proto";
+const RUST_V0_OUTPUT_DIR_PATH: &str = "crates/store/re_remote_store_types/src/v0";
+
+fn main() {
+    re_log::setup_logging();
+
+    let workspace_dir = Utf8Path::new(env!("CARGO_MANIFEST_DIR"))
+        .parent()
+        .and_then(|p| p.parent())
+        .and_then(|p| p.parent())
+        .unwrap();
+
+    assert!(
+        workspace_dir.join("CODE_OF_CONDUCT.md").exists(),
+        "failed to find workspace root"
+    );
```

Comment:
Don't leave future reader extremely confused :no_mouth: 

```suggestion
    // Check for something that only exists in root:
    assert!(
        workspace_dir.join("CODE_OF_CONDUCT.md").exists(),
        "failed to find workspace root"
    );